src/main/java/com/ibm/watson/developer_cloud/util/RequestUtils.java

```diff
@@ -159,25 +163,35 @@ public static String getUserAgent() {
     return userAgent;
   }
 
+  private static String loadSdkVersion() {
+    InputStream inputStream = RequestUtils.class.getClassLoader().getResourceAsStream("version.properties");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [In J2EE, getClassLoader() might not work as expected.  Use Thread.currentThread().getContextClassLoader() instead.](https://www.codacy.com/app/watson-developer-cloud/java-sdk/file/2730629156/issues/source?bid=3303699&fileBranchId=3349920#l167)